tests/unit/test_dynamodb.py

```diff
@@ -1,6 +1,7 @@
 import tempfile
```

Comment:
Should we only use the `use_threads` parameter on tests where the write operation is the main objective of the test? Just want to reduce the number of unnecessary tests but this is not major